export function handleError(res: any, error: any): void;
export {};
